soccer/Context.hpp

```diff
@@ -17,4 +18,6 @@ struct Context {
     SystemState state;
     GameState game_state;
     DebugDrawer debug_drawer;
+
+    std::vector<std::unique_ptr<VisionPacket>> vision_packets;
```

Comment:
@CollinAvidano, I think we discussed merging VisionReceiver into the VisionFilter module, and I'm now thinking that might be a good idea. That way we don't have to have this nasty vector of protobuf structures passed around and Context only contains "clean" data.

Whatever we do is pending moving logging stuff into a module, though.